src/template.html

```diff
@@ -139,17 +166,22 @@
 <section class="disclaimer">
 	<div class="o-grid-container">
 		<div class="o-grid-row">
-			<div class="disclaimer__left" data-o-grid-colspan="6">
+			<div class="disclaimer__left first-column" data-o-grid-colspan="2 S3">
 				<span class="disclaimer__paid_post">PAID POST</span>
-				<span class="disclaimer__sponsor">BY
-					<span class="disclaimer__sponsor-name">[[SPONSOR NAME HERE]]</span>
-				</span>
+        <span class="disclaimer__sponsor-wide">BY
+          <span class="disclaimer__sponsor-name">[[SPONSOR]]</span>
+        </span>
+<p></p>
+
 			</div>
-			<div data-o-grid-colspan="6">
-				<p class="disclaimer__sponsor-text">
+				<div class="disclaimer__sponsor-text second-column" data-o-grid-colspan="10 S7">
+            <span class="disclaimer__sponsor">BY
```

Comment:
I think we should hide one of these from screen readers (using [aria-hidden](https://msdn.microsoft.com/en-us/library/windows/apps/hh465705.aspx)), otherwise it will be read out twice in a row.